### PR TITLE
IP token refresh

### PR DESCRIPTION
**Pull request by skorpiom**
Hey,

This PR contains:

- changes on the backend related to deleting header x-auth-token and returning the token and the refresh token in cookies,
- two new endpoints 
           -  /refres-token returns new token and refresh token
           -  /logout blacklists refresh token and returns empty token and refresh token with lifespan set to 0
- interceptor on frontend that's in case of the invalidation of the token tries requests /refresh-token and repeat previous req

### CODE REVIEW

File: frontend/.env [hunk 1]
```diff
@@ -1,3 +1,3 @@
 SKIP_PREFLIGHT_CHECK=true
 
-REACT_APP_API_URL=https://coders-camp-organization-app.herokuapp.com
+REACT_APP_API_URL=http://localhost:5000
```
Comment (KrystianKjjk) [inline frontend/.env:3]: Damn

**Review by robert-czwartosz — COMMENTED** [review]
During runnning locally, when I try log in, I get error with message "Network Error".
Its very strange because, no request was sent after click Log In button.
@skorpiom @KrystianKjjk Have you any problems with log in locally?

**Review by KrystianKjjk — APPROVED** [review]

File: api/Src/Controllers/AuthController.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import * as express from 'express'
+import * as cache from 'memory-cache'
```
[the remark this answers is not in the document]
Comment (skorpiom) [reply api/Src/Controllers/AuthController.ts:2]: Hey,
I forgot to mention that .env file must be edited, probably that's the solution.
Please, let me know.

```
#jwt and jwt refresh (EXPIRESIN SET IN SECONDS ex. 900 NOT 15m (time is used to set cookies lifespan))
JWT_PRIVATE_KEY=jwtAccessTokenSecret
JWT_TOKEN_EXPIRESIN=900

JWT_REFRESH_PRIVATE_KEY=jwtSecretRefreshKey
JWT_REFRESH_EXPIRESIN=1800

```

**Review by robert-czwartosz — APPROVED** [review]

**Comment by robert-czwartosz** [comment]
@skorpiom After resolving conflicts, feel free to merge

**Pull request merged**